sdk/python/feast/repo_operations.py

```diff
@@ -107,15 +107,20 @@ def parse_repo(repo_root: Path) -> ParsedRepo:
         for attr_name in dir(module):
             obj = getattr(module, attr_name)
             if isinstance(obj, FeatureTable):
-                res.feature_tables.append(obj)
+                if obj.defined_in is not None and obj.defined_in == module.__file__:
```

Comment:
Can module.__file__ be None ?

Reply:
we should maybe also raise exceptions in case defined_in is None

Reply:
Done. And no, module.__file__ cannot be None